test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -547,24 +547,22 @@ withPact
     -> IO BlockHeaderDb
     -> MemPoolAccess
     -> IO FilePath
-    -> (IO (TQueue RequestMsg) -> TestTree)
+    -> (IO PactQueue -> TestTree)
     -> TestTree
 withPact version logLevel iopdb iobhdb mempool iodir f =
     withResource startPact stopPact $ f . fmap snd
   where
     startPact = do
         mv <- newEmptyMVar
-        reqQ <- atomically newTQueue
+        reqQ <- atomically $ newTBQueue pactQueueSize
         pdb <- iopdb
         bhdb <- iobhdb
         dir <- iodir
-        a <- withLink $ initPactService version cid logger reqQ mempool mv
+        a <- async $ initPactService version cid logger reqQ mempool mv
```

Comment:
This isn't the only place `withLink` was used - we can remove it from `Chainweb.Utils`, and there are 2 more tests using it (`PactReplayTest` and `PactInProcApiTest` iirc)